### ISSUE TITLE
execution_id missing in PubSub cloud function

### ISSUE THREAD

**Issue by techieshark** [googleapis/nodejs-logging-winston#465]
Is there something this library, or the users of it, could do to get a pubsub event id to be included in the one-line summary seen in Stackdriver? As described below, I've tried to set it manually as a label, but that seems to not impact the summary line.

#### Environment details

  - OS: cloud-functions 
  - Node.js version: 10
  - `@google-cloud/logging-winston` version: 3.0.5

#### Steps to reproduce

  1. Create pubsub cloud function which takes a `context` as second parameter.
  2. Log with winston to stackdriver transport (as per the docs), and try manually setting `labels["execution_id"]` to the event id:

```
 logger.debug('Starting processing pubsub message', {
    labels: {
      // Observe console.log entries to see that an event ID is the execution_id.
      // We manually populate it so we can see the group of all logs for a
      // single execution/event.
      execution_id: context.eventId,
    }
}
```

<img width="1154" alt="image" src="https://user-images.githubusercontent.com/1072292/75304994-e8aac380-5898-11ea-8eb3-d10dea2018bc.png">

Expected: the one line (non-expanded) summary includes the execution id / event id. This is the case with console.log.

Actual: non is present. See screenshot.

**Comment by dfdgsdfg** [comment]
https://github.com/googleapis/nodejs-logging/issues/591

https://github.com/firebase/firebase-functions/issues/560

**Comment by techieshark** [comment]
Solution might be using firebase-functions logger instead (for cloud functions):

https://github.com/firebase/firebase-functions/issues/560#issuecomment-640528561

https://github.com/firebase/firebase-functions/releases/tag/v3.7.0

>  Adds functions.logger SDK to enable structured logging in the Node.js 10 runtime. For example:

> const functions = require('firebase-functions');

> functions.logger.debug('example log with structured data', {
>  uid: user.uid,
>  authorized: true,
> });

Internally this just is like 
```
console.log(`your-message-line and then ${JSON.stringify(structuredInfo)}`);
```
which is supposed to work on Node >= 10

I'll try this soon.

**Comment by nicoleczhu** [comment]
Hi @techieshark , a few notes that may help: 

1. **Getting exec_id to show up as a Summary Field (the UI bubbles)**: You can manually configure summary fields yourself [steps here](https://cloud.google.com/logging/docs/view/logs-viewer-interface#add_summary_fields). Though to your point, we did expect this to be the default behavior. We've flagged this internally, so thanks for informing us! 

2. **Getting execution_id by default**: this is blocked by [this upstream issue](https://github.com/GoogleCloudPlatform/functions-framework/issues/34). Will follow up on it for you. After that’s in, we can work on getting everyone trace/exec ids by default, minimally for pubsub/scheduler triggered cloud functions. [tracked here](https://github.com/googleapis/nodejs-logging/issues/591). 

Closing the issue here since the fix is in different repos and already tracked and tagged to this issue instance.

**Comment by techieshark** [comment]
Thanks for the tips, @nicoleczhu! Looking forward to seeing those upstream issues resolved - would be quite helpful for our logging ux.
